core/src/main/java/hudson/Main.java

```diff
@@ -220,7 +220,8 @@ private static HttpURLConnection open(URL url) throws IOException {
     public static boolean isUnitTest = false;
 
     /**
-     * Set to true if we are running inside {@code mvn hpi:run} or {@code mvn jetty:run}.
+     * Set to true if we are running inside {@code mvn jetty:run}.
+     * This is also set if running inside {@code mvn hpi:run} since plugins parent POM 2.30.
```

Comment:
does this help?, that version is ancient?

Reply:
I don't remember what I did to encounter this problem but it is the reason I chose to do https://github.com/jenkinsci/jenkins/pull/4630/files#diff-841df66889e3c3214c68de134dfe05beR2046